### PR TITLE
Refresh cookiecutter template using cruft

### PR DESCRIPTION
**Pull request by tlvu**
This PR is part of issue https://github.com/bird-house/cookiecutter-birdhouse/issues/61, rolling out cruft as a mechanism to easily update cookiecutter template changes to existing project.

Requires PR https://github.com/bird-house/cookiecutter-birdhouse/pull/87 and https://github.com/bird-house/cookiecutter-birdhouse/pull/88.

Other noticeable changes in this PR:
* fix weird version format cb63d07e6cfef52747d3b6f427ca9cef9a973933
* fix flakes8 failure E741 ambiguous variable name 'l' 76345b93c783998157f32012f02bae8378514c7b + 7a312f100025d7a61ea964dbe15092258f146871

RtD build passed with list of processes https://readthedocs.org/projects/finch/builds/11131993/

### CODE REVIEW

**Comment by huard** [comment]
@tlvu RtD: Done

File: Makefile [hunk 1]
```diff
@@ -4,6 +4,11 @@ APP_NAME := finch
 FINCH_WPS_URL := http://localhost:5000
```
Comment (huard) [inline Makefile:4]: Create PR to replace FINCH_WPS_URL by WPS_URL in the code and notebooks.

Comment (tlvu) [reply Makefile:4]: I'll handle in this PR together with  TravisCI and possibly RtD.

**Comment by tlvu** [comment]
Oh just noticed Finch build is on travis-ci.org while Raven is on travis-ci.com ... why do we have the difference?

**Comment by MatProv** [comment]
> @MatProv are you replacing David Caron on CRIM side? Should I be including you in all future Finch changes? Thanks.

@tlvu Yes please include me in future changes. I want to stay informed of changes on Finch, even if I occasionally work on it. Thanks

**Comment by huard** [comment]
@tlvu No idea about Travis.

File: finch/processes/subset.py [hunk 2]
```diff
@@ -37,13 +37,13 @@ def finch_subset_gridpoint(
...
     try:
-        longitudes = [float(l) for l in lon_value.split(",")]
+        longitudes = [float(lon) for lon in lon_value.split(",")]
     except AttributeError:
         longitudes = [float(lon_value)]
 
     lat_value = request_inputs[wpsio.lat.identifier][0].data
     try:
-        latitudes = [float(l) for l in lat_value.split(",")]
+        latitudes = [float(lat) for at in lat_value.split(",")]
```
Comment (MatProv) [inline finch/processes/subset.py:46]: typo, 'lat'

Comment (tlvu) [reply finch/processes/subset.py:46]: Indeed, I introduced a typo when fixing flakes8.  Fixed.

**Comment by tlvu** [comment]
Can I be admin on this Finch repo?  Looks like only admin can re-trigger TravisCI builds (python 3.7 env failed while 3.8 passed !)

Also I fixed a weird version format in commit cb63d07 but the weird format has existed for a while, so I wonder if we actually wanted this weird format.

**Comment by tlvu** [comment]
RtD build passed: https://readthedocs.org/projects/finch/builds/11099314/

**Comment by huard** [comment]
@tlvu Done
Version thing does not ring a bell.

**Comment by tlvu** [comment]
Oh crap the processes list is empty: https://pavics-sdi.readthedocs.io/projects/finch/en/refresh-cookiecutter-template-using-cruft/processes.html, will look at this next week.

**Comment by tlvu** [comment]
Humm a bunch of "WARNING: autodoc: failed to import process" but RtD build still passed !  That's bad !  Raven have the same problem (empty processes).

**Comment by tlvu** [comment]
Finally RtD build passed with list of processes https://readthedocs.org/projects/finch/builds/11131993/

**Pull request merged**